web-app/src/pages/Login/LoginPage.tsx

```diff
@@ -59,6 +63,24 @@ type AuthCompProps = {
 	isConfirmButtonVisible: boolean;
 };
 
+const requestLogin = async (username: string, password: string, signup: boolean): Promise<boolean> => {
+	const rawResponse = await fetch( API_ROUTE + '/auth/' + (signup ? "signup" : "login"), {
+	  method: 'POST',
```

Comment:
The indentation is broken

Reply:
## Fix

Identation is fixed with Prettier.